src/psyclone/gocean1p0.py

```diff
@@ -1468,6 +1468,26 @@ def scalars(self):
         args = args_filter(self._args, arg_types=["scalar"])
         return [arg.name for arg in args]
 
+    def append(self, name):
+        ''' Append a GOKernelArgument to the Argument list.
```

Comment:
"Create and append a GOKernelArgument..."

Reply:
Done